docs/UserGuide.md

```diff
@@ -58,121 +33,47 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 </div>
 
-### Viewing help : `help`
-
-Shows a message explaning how to access the help page.
-
-![help message](images/helpMessage.png)
-
-Format: `help`
```

Comment:
Perhaps we can add this back, since we are implementing `help` in v1.2
But we would need to update the image to https://ay2021s1-cs2103-t16-2.github.io/tp/UserGuide.html